interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/*
+ *  Daily Summary Report. (/interface/reports/daily_summary_report.php
+ *  
+ *
+ *  This report shows date wise numbers of the Appointments Scheduled,
+ *  New Patients, Visited patients, Total Charges, Total Co-pay and Balance amount for the selected facility.
+ * 
+ * Copyright (C) ....
+ * 
+ * LICENSE: This program is free software; you can redistribute it and/or 
+ * modify it under the terms of the GNU General Public License 
+ * as published by the Free Software Foundation; either version 3 
+ * of the License, or (at your option) any later version. 
+ * This program is distributed in the hope that it will be useful, 
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+ * GNU General Public License for more details. 
+ * You should have received a copy of the GNU General Public License 
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;. 
+ * 
+ * @package OpenEMR 
+ * @author Rishabh Software
+ * @link http://www.open-emr.org 
+ *
+ */
+
+$fake_register_globals = false;
+$sanitize_all_escapes = true;
+
+require_once("../globals.php");
+require_once "$srcdir/options.inc.php";
+require_once "$srcdir/appointments.inc.php";
+
+
+$selectedFromDate = isset($_POST['form_from_date'])?$_POST['form_from_date']:date('Y-m-d'); // From date filter
+$selectedToDate = isset($_POST['form_to_date'])?$_POST['form_to_date']:date('Y-m-d');     // To date filter
+$selectedFacility = isset($_POST['form_facility'])?$_POST['form_facility']:"";  // facility filter
+$selectedProvider = $_POST['form_provider'];  // provider filter
+
+$from_date = fixDate($selectedFromDate, date('Y-m-d'));
+$to_date = fixDate($selectedToDate, date('Y-m-d'));
+?>
+
+<html>
+    <head>
+        <?php html_header_show(); ?>
+        <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+        <script type="text/javascript" src="../../library/textformat.js"></script>
+        <script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
+        <script type="text/javascript">
+
+
+            var mypcc = '<?php echo $GLOBALS['phone_country_code'] ?>';
+
+            function submitForm() {
+                var fromDate = $("#form_from_date").val();
+                var toDate = $("#form_to_date").val();
+
+                if (fromDate === '') {
+                    alert("Please select From date");
+                    return false;
+                }
+                if (toDate === '') {
+                    alert("Please select To date");
+                    return false;
+                }
+                if (Date.parse(fromDate) > Date.parse(toDate)) {
+                    alert("From date should be less than To date");
```

Comment:
We have used xls function to translate text in the Javascript.
Please see the changes at pull request https://github.com/bradymiller/openemr/pull/9